.github/workflows/ci.yaml

```diff
@@ -35,6 +35,8 @@ jobs:
       ts: ${{ steps.filter.outputs.ts }}
       k8s: ${{ steps.filter.outputs.k8s }}
       ci: ${{ steps.filter.outputs.ci }}
+      offlinedocs-only: ${{ steps.filter.outputs.offlinedocs_count == steps.filter.outputs.all_count }}
+      offlinedocs: ${{ steps.filter.outputs.offlinedocs }}
```

Comment:
This will skip lint and format when we only target offlinedocs.
lint and fmt checks do not check offline docs so this was useless here